sass/_imports.sass

```diff
@@ -10,7 +10,7 @@ $prefix-for-opera: false
 // Incuna-sass
 @import incuna-sass/functions
 @import incuna-sass/normalize
-// @import incuna-sass/reset
+@import incuna-sass/incuna-normalize
```

Comment:
reset didn't offer any real benefit here, it'll likely be added to individual projects anyway so it's not essential to add stuff here.

Incuna normalize is a new file we made with sass we tend to add to every project, so we don't always have to repeat ourselves, things like `box-sizing: border-box`